### ISSUE TITLE
Old jQuery dep

### ISSUE THREAD

**Issue by TrySound** [digitalBush/jquery.maskedinput#291]
Why maskedinput depends on jQuery <2?
It works fine with the newest version.

**Comment by FagnerMartinsBrack** [comment]
Maybe to make sure it works in the browsers jQuery < 2 supports. jQuery 2 does not support IE 8, for instance.

**Comment by TrySound** [comment]
I use bower and gulp for building. And main-bower-files can't use jQuery > 2 with old dep in plugin.
So, I just need to delete <2 in bower.json.

**Comment by digitalBush** [comment]
There's nothing stopping the plugin from working on 2.0. I think we can safely delete that from the bower.json.

**Comment by TrySound** [comment]
Thanks a lot!

**Comment by RQuadling** [comment]
Still getting the restricted upper limit.

```
bower cache clean
bower install jquery.maskedinput --save --force-latest
```

is giving me ...

```
...
bower deleted       Cached package jquery.maskedinput: C:\Users\RichardQ\AppData\Roaming\bower\cache\packages\68e7d843c2125f1861fa2b4cfce398b0\1.4.0
...
bower jquery.maskedinput#*  not-cached git://github.com/digitalBush/jquery.maskedinput.git#*
bower jquery.maskedinput#*     resolve git://github.com/digitalBush/jquery.maskedinput.git#*
bower jquery.maskedinput#*    download https://github.com/digitalBush/jquery.maskedinput/archive/1.4.0.tar.gz
bower jquery.maskedinput#*     extract archive.tar.gz
bower jquery.maskedinput#*     invalid-meta jquery.maskedinput is missing "main" entry in bower.json
bower jquery.maskedinput#*         resolved git://github.com/digitalBush/jquery.maskedinput.git#1.4.0
bower jquery#>=1.8.3 <2.0        not-cached git://github.com/jquery/jquery.git#>=1.8.3 <2.0
bower jquery#>=1.8.3 <2.0           resolve git://github.com/jquery/jquery.git#>=1.8.3 <2.0
bower jquery#>=1.8.3 <2.0          download https://github.com/jquery/jquery/archive/1.11.2.tar.gz
bower jquery#>=1.8.3 <2.0           extract archive.tar.gz
bower jquery#>=1.8.3 <2.0          resolved git://github.com/jquery/jquery.git#1.11.2

Please note that,
    jquery.maskedinput#1.4.0 depends on jquery#>=1.8.3 <2.0 which resolved to jquery#1.11.2
    DigiPoint depends on jquery#~2.1.3 which resolved to jquery#2.1.3
Resort to using jquery# which resolved to jquery#2.1.3
Code incompatibilities may occur.

bower jquery                     resolution Saved jquery#~2.1.3 as resolution
```

Bower is getting the released file.

Can you re-release 1.4.0 or tag the current version as 1.4.1.

Thank you.
